simexpal/base.py

```diff
@@ -725,6 +766,33 @@ def output_file_path(self, ext):
 		return os.path.join(self.experiment.output_subdir,
 				get_output_file_name(ext, self.instance.filename, self.repetition))
 
+	def get_status(self):
+		in_submission = os.access(self.aux_file_path('lock'), os.F_OK)
+		submitted = os.access(self.aux_file_path('run'), os.F_OK)
+		started = os.access(self.output_file_path('out'), os.F_OK)
+		finished = os.access(self.output_file_path('status'), os.F_OK)
```

Comment:
Instead of always accessing all files, we should access them only when necessary (for performance reasons: `os.access` needs to perform filesystem I/O and is relatively expensive).

Reply:
Done.